assets/stylesheets/kitten/components/form/_radio-button-set.scss

```diff
@@ -0,0 +1,40 @@
+/// Radio button set
+///
+/// @group form
+///
+/// @example scss - usage
+///
+///   @include k-RadioButtonSet;
+///
+/// @example html
+///
+///   <div class="k-RadioButtonSet">
+///     <div class="k-RadioButton k-RadioButtonSet__radioButton">
+///       <input id="karl-radio-button-1"
+///              type="radio"
+///              name="karl-radio-button"
+///              class="k-RadioButton__input" />
+///       <label for="karl-radio-button-1" class="k-RadioButton__label">
+///         Default
+///       </label>
+///     </div>
+///
+///     <div class="k-RadioButton k-RadioButtonSet__radioButton">
+///       <input id="karl-radio-button-2"
+///              type="radio"
+///              name="karl-radio-button"
+///              class="k-RadioButton__input" />
+///       <label for="karl-radio-button-2" class="k-RadioButton__label">
+///         Default
+///       </label>
+///     </div>
+///   </div>
+
+@mixin k-RadioButtonSet {
```

Comment:
> Le vote était sur slack #the-name-game :p.

Oui je parlais du nombre de votes sur Slack, justement https://kisskiss.slack.com/archives/the-name-game/p1489676389477947

> c'est pas mal dans la mesure ou la checkbox et le radio button implique un choix :). 

Un `<select>` ou un `<input>` ça n'implique pas un choix ? ;)